apps/src/dance/Dance.js

```diff
@@ -176,6 +268,15 @@ Dance.prototype.reset = function () {
 
   this.nativeAPI.reset();
   this.p5.noLoop();
+
+  var softButtonCount = 0;
+  for (var i = 0; i < this.level.softButtons.length; i++) {
+    document.getElementById(this.level.softButtons[i]).style.display = 'inline';
+    softButtonCount++;
+  }
+  if (softButtonCount) {
+    $('#soft-buttons').removeClass('soft-buttons-none').addClass('soft-buttons-' + softButtonCount);
```

Comment:
Nit: this has an implicit dependency on `window.$` (vs. an explicit `import` statement).  Worth writing this without jQuery?